readme.md

```diff
@@ -23,8 +23,16 @@ Get location data of any IP Address within seconds.
 
 ## 🎩 INSTALLATION
 
-Install the CLI globally. If you are windows user, run your command prompt/terminal as administrator. If you are macOS user, add `sudo` before the following command.
+Install the CLI globally. If you are windows user, run your command prompt/terminal as administrator.
 
+```
+npm i -g geo-location-cli
+```
+For Linux & macOS users
+```
+sudo su
```

Comment:
@IahmadDev don't you think it would be better to add this before adding the installation command. Otherwise, the user may skip it & do the installation first. I think it would probably be best if you explain it in the first line after the `as administrator`

Reply:
Also, please make sure you explain the commands separately. Like `sudo` for macOS & `su` for Linux. Adding them together will confuse the user. You can also just add `sudo` for both macOS & Linux. It works for both of these operating systems.

Reply:
Sure let me get it right again

Reply:
@iahmadDev have you fixed it?

Reply:
Yes I have